### ISSUE TITLE
Not able to properly log to Sentry within middlewares

### ISSUE THREAD

**Issue by nazarov-tech** [huge-success/sanic#1141]
When you try this in middleware:

```python
logger.info(f'Request to {request.url} occured', extra=extra)
```

this happens:

![image1](https://i.imgur.com/u6VUkhH.png)

which leads to `<unlabeled event>` in Sentry:

![image2](https://i.imgur.com/MFUBDES.png)

As I can understand `sanic.access` in middlewares ignores existing logger configuration.

So you have to do things like this:

```python
raven_client.captureMessage(message=f'Request to {request.url} occurred', data={
        'request': {
            'url': request.url,
            'data': {},
            'query_string': '...',
            'method': 'POST',
        },
        'logger': 'sanic.access',
}, level='info', extra=extra)
```

..which is better, but you would still also get `<unlabeled event>` (apparently, `sanic.access` itself makes logging calls).

Or should I address this issue to the [raven](https://github.com/getsentry/raven-python)?

**Comment by madisvain** [comment]
@nazarov-tech Raven has released Sanic support in 6.7.0 so maybe you can check it out:
https://github.com/getsentry/raven-python/blob/master/raven/contrib/sanic.py

They also have automatic logging configuration - I have not tested this personally but just to let you know.

**Comment by nazarov-tech** [comment]
@madisvain thanks I will look it up
